tests/unit/mixins/versionable.tests.js

```diff
@@ -0,0 +1,28 @@
+describe("Space.messaging.Versionable", function() {
+
+  const MyVersionableClass = Space.Object.extend({
+    mixin: Space.messaging.Versionable,
+    schemaVersion: 3,
+    migrateFromVersion1(data) { data.first = 'first'; },
```

Comment:
With this version of code, if i remove this line test fails with `- exception - message expected undefined to equal 'first'`